hrjobcontract/CRM/Hrjobcontract/Upgrader.php

```diff
@@ -1117,9 +1167,7 @@ private function removePensionTypeOptionGroup(&$messages) {
         'name' => 'hrjc_pension_type',
       ]);
 
-      if ($pensionOptionGroup['count'] == 0) {
-        $messages[] = 'Error deleting hrjc_pension_type option group: ' . $e->getMessage();
-      } else {
+      if ($pensionOptionGroup['count'] > 0) {
```

Comment:
with this, you're not using $messages in this method anymore. Please remove the param from the method